explainer.md

```diff
@@ -77,5 +184,32 @@ domain of the web app include this extension. Browser would refuse to
 load web apps without Source Code Transparency from a domain using a
 certificate with this extension.
 
+```
+# jka commentary
+This seems OK, although I'd note that many web clients don't use Transport
+Layer Security (TLS - also known as HTTPS), required for X.509 certificates.
```

Comment:
So - one of the changes I made in this proposal is to separate the Source Code Transparency log from the Certificate Transparency log, and use the latter only to signal that the former should be consulted. But, other mechanisms to signal the same thing could be added in the future as well.

That being said, for web apps that care about security, I think using TLS is quite essential.

Reply:
The typical example I think of here is a banking app.  I'd personally like the download process for my banking app to be integrity-guaranteed and authentic but _not_ confidential.

However while using the app I'd _probably_ like most transactions to have all three of confidentiality, integrity and authenticity.  Although to be honest sometimes it's worthwhile to know that different people pay the same prices for the same goods (again, not a normal threat to consider in most civilized countries, but a theoretical one).